packages/client/src/views/SysAdmin/Team/user/userCreation/CreateNewUser.tsx

```diff
@@ -82,6 +83,11 @@ const Container = styled.div`
 class CreateNewUserComponent extends React.Component<Props & IDispatchProps> {
   async componentDidMount() {
     const { userId, client } = this.props
+    if (
+      this.props.match.path.includes(CREATE_USER_ON_LOCATION.split('/:')[0])
```

Comment:
@sadmananik could you please tell me if I am wrong about what this check does? Does this block check whether the current path contains `createUserInLocation` ? If so, this block clears data if navigated to the first step of create user form... If the user enters form data, closes it with the cross button from the first step, and then presses back again, then the user can see empty form data. I see no problem there, but you can still verify if that's the expected behavior.